examples/asa/assets/5-contract-owned-asa.py

```diff
@@ -0,0 +1,62 @@
+# Escrow Account Contract
+
+# Add parent directory to path so that algobpy can be imported
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parseArgs
+from pyteal import *
+
+def contract_asa(app_id):
+
+    # verify neither transaction
+    # contains a rekey
+    rekey_check = And(
+        Gtxn[0].rekey_to() == Global.zero_address(),
+        Gtxn[1].rekey_to() == Global.zero_address()
+    )
+
+    pay = And(
+        # The first transaction must be 
+        # an ApplicationCall (ie call stateful smart contract)
+        Gtxn[0].type_enum() == TxnType.ApplicationCall,
+
+        # The specific App ID must be called
+        # This should be changed after creation
+        Gtxn[0].application_id() == Int(app_id),
+
+        Gtxn[1].type_enum() == TxnType.AssetTransfer,
+        Gtxn[1].amount() <= Int(100),
```

Comment:
So this is an inforced condition as well (along with max 1 asa creation) ?  Could you please add in comment at the top of file (like we do in approval.py)?